apps/projects/templates/meinberlin_projects/includes/timeline.html

```diff
@@ -35,25 +35,48 @@
 
 {% if not object|is_offlineevent %}
 <div class="l-center-6">
-    {% for phase in project.phases %}
-        <div class="phase-info {% if phase = active_phase %}phase-info--active{% endif %}">
-
+    <div class="phase-info">
+        <div class="phase-info__active">
             <h3 class="phase-info__title">
-                {{ phase.name }}
+                {{ active_phase.name }}
             </h3>
             <div class="phase-info__subtitle">
-                {{ phase.start_date }}
-                &mdash; {{ phase.end_date }}
+                {{ active_phase.start_date|date:'DATE_FORMAT' }}
+                &ndash; {{ active_phase.end_date|date:'DATE_FORMAT' }}
             </div>
             <div class="phase-info__description">
-                {{ phase.description }}
+                {{ active_phase.description }}
             </div>
         </div>
-    {% empty %}
-        <h3 class="phase-info__title">
-            {% trans "Participation is not possible at the moment." %}
-        </h3>
-    {% endfor %}
+        {% if project.phases|length > 1 %}
+            <div class="phase-info__overview collapse" id="phase-overview">
+                {% for phase in project.phases %}
+                    <div class="phase-info__item">
+                        <h3 class="phase-info__title{% if phase == active_phase %} lr-bar{% endif %}">
+                            {% if phase == active_phase %}
+                                <span class="lr-bar__left">{{ phase.name }}</span>
+                                <span class="lr-bar__right phase-info__highlight">{% trans 'active' %}</span>
+                            {% else %}
+                                {{ phase.name }}
+                            {% endif %}
+                        </h3>
+                        <div class="phase-info__subtitle">
+                            {{ phase.start_date|date:'DATE_FORMAT' }}
+                            &ndash; {{ phase.end_date|date:'DATE_FORMAT' }}
+                        </div>
+                    </div>
+                {% empty %}
+                    <h3 class="phase-info__title">
+                        {% trans "Participation is not possible at the moment." %}
+                    </h3>
+                {% endfor %}
+            </div>
+            <button class="button button--link" type="button" data-toggle="collapse" data-target="#phase-overview" aria-expanded="false" aria-controls="phase-overview">
+                <span class="u-if-collapsed">{% trans 'Show all phases' %}</span>
+                <span class="u-if-not-collapsed">{% trans 'Hide all phases' %}</span>
+            </button>
+        {% endif %}
+    </div>
```

Comment:
The case where there is not `active_phase` is missing. We need to be careful with wording in that one because it can be before, after, or even in between phases.

@damusp suggested to automatically generate a text that summarizes the process so far. But I think this is out of scope for now.